controllers/imageclusterinstall_controller_test.go

```diff
@@ -823,7 +823,7 @@ var _ = Describe("Reconcile", func() {
 	})
 
 	It("configures a referenced BMH", func() {
-		bmh := bmhInState(bmh_v1alpha1.StateAvailable)
+		bmh := bmhInState(bmh_v1alpha1.StateExternallyProvisioned)
```

Comment:
Are we expecting people to configure the BMH with externally provisioned now?
Will that disrupt inspection?

Reply:
Yes, when the user configures the BMH with externally provisioned inspection will be disabled